src/Slack/MessageSender.php

```diff
@@ -172,6 +184,8 @@ public function sendAdminMessage(SecretSanta $secretSanta, string $code, string
 
 Remember, with great power comes great responsibility!
 
+The event will happen at this time : ' . date('H:i - m/d/Y', $secretSanta->getOptions()['scheduled_at']) . ' UTC
```

Comment:
This wording is not accurate :thinking: 

The sentence should only be added if there is a scheduling, and it should say "the messages will be sent at", not "this event" IMO.

Reply:
I agree and I changed it. Thanks for the feedback